.coveragerc

```diff
@@ -1,6 +1,7 @@
 [run]
-disable_warnings = no-data-collected
+data_file = .coverage-reports/.coverage
 branch = True
+parallel = True
```

Comment:
Using this option is obligatory for running tests in parallel. It will create _many_ files like `.coverage.{hostname}.23098.826306`, which will be combined by `coverage combine` later. Following up on this, I think `--parallel` can be removed from `tox.ini`.

Reply:
> I think using the `--parallel` option within `tox.ini` can be removed.

Implemented with 290ffc15f3.